website/docs/getting-started/setup/installation-guides/kubernetes/README.mdx

```diff
@@ -11,72 +11,99 @@ This page provides steps to install Appsmith on a Kubernetes cluster using the [
 
 <VideoEmbed host="youtube" videoId="wZzYL1uZwds" title="How To Deploy Appsmith On Kubernetes" caption="How To Deploy Appsmith On Kubernetes" />
 
-### Prerequisites
-1. Ensure that you have at least 15 GB of free storage space and 4 GB of RAM.
-2. [Helm Package Manager](https://helm.sh/docs/intro/install/) to install and manage applications on Kubernetes clusters.
-3. [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl) to manage resources on Kubernetes clusters.
-4. Follow the steps below to install Appsmith on your Kubernetes cluster:
-
-<Tabs groupId="configurations" queryString="current-configuration">
- <TabItem label="AWS EKS" value="AWSEKS">
-
-  a. Set up the Kubernetes cluster by [creating a kubeconfig for Amazon EKS](https://docs.aws.amazon.com/eks/latest/userguide/create-kubeconfig.html).
-
-  b. Ensure that you have a default storage class running. If not, [create default storage class](https://docs.aws.amazon.com/eks/latest/userguide/storage-classes.html).
+## Prerequisites
+1. At least 15 GB of free storage space.
+2. 4 GB of RAM.
+
+## Before you begin
+1. Installed Helm package manager on your local machine. See the official Helm documentation for your [operating system](https://helm.sh/docs/intro/install/#through-package-managers).
+2. Installed and configured `kubectl` to interact with your Kubernetes cluster. Follow the below guides available on the official Kubernetes documentation for instructions on how to install `kubectl` on your specific operating system: 
+    * [MacOS using Homebrew](https://kubernetes.io/docs/tasks/tools/install-kubectl-macos/#install-with-homebrew-on-macos)
+    * [Linux using native package management](https://kubernetes.io/docs/tasks/tools/install-kubectl-linux/#install-using-native-package-management)
+    * [Windows using Chocolatey, Scoop, or winget](https://kubernetes.io/docs/tasks/tools/install-kubectl-windows/#install-nonstandard-package-tools).
+3. A running Kubernetes cluster. If not present, follow these steps to set up the cluster on AWS EKS:
+
+  :::info
+   You are free to choose your preferred platform for hosting the Kubernetes cluster. Make sure you have configured Kubeconfig and created a default storage class before proceeding with the Appsmith installation. If you face any issues, reach out to the support team at [support@appsmith.com](mailto:support@appsmith.com).
```

Comment:
we are recommending chat widget everwhere right?

Reply:
Yes, but here it makes sense for them to contact support using email, as they are having issues in their prerequisite setup. What do you think?

Reply:
either way its going to be answered by Pranav and team only. The reason I'm thinking we should use the chat widget is to be able to easily measure how many people have issues and reach out from docs. Secondly its one less step for them to send an email